README.Rmd

```diff
@@ -277,93 +93,22 @@ library("crul")
 
 ## Configuration
 
-We set the following defaults:
-
-* `dir` = "."
-* `record` = "once"
-* `match_requests_on` = `c("method", "uri")`
-* `allow_unused_http_interactions` = `TRUE`
-* `serialize_with` = `"yaml"`
-* `persist_with` = `"FileSystem"`
-* `ignore_hosts` = `NULL`
-* `ignore_localhost` = `FALSE`
-* `uri_parser` = `crul::url_parse`
-* `preserve_exact_body_bytes` = `FALSE`
-* `turned_off` = `FALSE`
-* `ignore_cassettes` = `FALSE`
-* `re_record_interval` = `NULL`
-* `clean_outdated_http_interactions` = `NULL`
-* `cassettes` = `list()`
-* `linked_context` = `NULL`
-* `vcr_logging` = `"vcr.log"`
-* `vcr_logging_opts` = `list()`
-
-
-You can get the defaults programmatically with
-
-```r
-vcr_config_defaults()
-```
-
-You can change all the above defaults with `vcr_configure()`:
-
-```r
-vcr_configure()
-```
-
-Calling `vcr_configuration()` gives you some of the more important defaults in a nice tidy print out
-
-```r
-vcr_configuration()
+```{r child='man/rmdhunks/configuration.Rmd'} 
 ```
 
-
-<!-- `use_cassette()` is an easier approach. An alternative is to use
-`insert_cassett()` + `eject_cassette()`.
-
-`use_cassette()` does both insert and eject operations for you, but
-you can instead do them manually by using the above functions. You do have
-to eject the cassette after using insert. -->
-
 ## Matching/Matchers
 
-`vcr` looks for similarity in your HTTP requests to cached requests. You
-can set what is examined about the request with one or more of the
-following options:
-
-* `body`
-* `headers`
-* `host`
-* `method`
-* `path`
-* `query`
-* `uri`
-
-By default, we use `method` (HTTP method, e.g., `GET`) and `uri` (test for exact match against URI, e.g., `http://foo.com`).
-
-You can set your own options by tweaking the `match_requests_on` parameter:
-
-```{r echo=FALSE}
-unlink(file.path(cassette_path(), "one.yml"))
-```
-
-```{r eval = FALSE}
-use_cassette(name = "one", {
-    cli$post("post", body = list(a = 5))
-  },
-  match_requests_on = c('method', 'headers', 'body')
-)
+```{r child='man/rmdhunks/matching.Rmd'} 
 ```
 
 ## vcr in other languages
 
 The canonical `vcr` (in Ruby) lists ports in other languages at <https://github.com/vcr/vcr>
 
-## NOTE
+## Note about missing features
 
-There's a number of features in this package that are not yet supported, but for which their parameters are found in the package. For example, `decode_compressed_response` is a parameter in `use_cassette()` but it is ignored right now.
-
-We've tried to make sure the parameters that are ignored are marked as such. Keep an eye out for package updates for changes in these parameters, and/or let us know you want it and we can move it up in the priority list.
+```{r child='man/rmdhunks/missing-features.Rmd'} 
```

Comment:
can you remove the reference to `decode_compressed_response` as I'm dropping that setting  https://github.com/ropensci/vcr/issues/30 - just mention some unimplemented features in general from Ruby vcr

Reply:
Done